CHANGELOG.md

```diff
@@ -5,6 +5,14 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Unreleased
+
+### Fixed
+- [ISSUE-101](https://github.com/microsoft/scitt-ccf-ledger/issues/101): Replaced workaround code with `t_cose` for signature verification of Notary style claims.
```

Comment:
Please stick to existing format of changelog entries. When `#xy` is copied to a release entry it is automatically linked.